app/grandchallenge/components/models.py

```diff
@@ -962,8 +962,43 @@ def __str__(self):
         else:
             return f"Component Interface Value {self.pk} for {self.interface}"
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._value_orig = self.value
+        self._image_orig = self.image
+        self._file_orig = self.file
+
+    def save(self, *args, **kwargs):
+        if (
+            (
+                self._value_orig not in (None, self.interface.default_value)
+                and self.value is not None
```

Comment:
Wouldn't this allow you to still set the value to `None`?

Reply:
Yes, that's correct. That is currently necessary for  `civ_value_to_file()` to keep functioning.

Reply:
Ah that makes sense then :slightly_smiling_face: